code/_onclick/telekinesis.dm

```diff
@@ -104,7 +104,7 @@
 /obj/item/tk_grab/afterattack(atom/target , mob/living/user, proximity, params)//TODO: go over this
 	if(!target || !user)
 		return
-	if(last_throw+3 > world.time)
+	if(last_throw + 3 SECONDS > world.time)
```

Comment:
make this a define at the top of the file, and undef it at the bottom of the file.